alexandria/storages/fields.py

```diff
@@ -28,7 +25,9 @@ class DynamicStorageFileField(models.FileField):
     # of Alexandria: https://github.com/projectcaluma/alexandria/issues/480
 
     def __init__(self, storage=None, **kwargs):
-        storage = storages.create_storage({"BACKEND": settings.ALEXANDRIA_FILE_STORAGE})
+        self.storage = storages.create_storage(
+            {"BACKEND": settings.ALEXANDRIA_FILE_STORAGE}
+        )
         super().__init__(storage=storage, **kwargs)
```

Comment:
that's a bit hard to grasp. `storage` is not handled here but `self.storage` is. aka in the `super()` sth sth statement the storage arg is passed on or always `None` or the default is `None` which we clearly don't want, right? How are conflicting values of `self.storage` and `super().__init__(storage=storage)` behave

Reply:
This is a mistake, reverted